README.md

```diff
@@ -123,20 +123,67 @@ Note: a single benchmark for FIO will take about 6 minutes to finish.
 
 See [./deploy/fio.yaml](https://github.com/yasker/benchmark/blob/main/deploy/fio.yaml) for available options.
 
-#### Deploy Comparison Benchmark in Kubernetes cluster
+### Deploy Single Volume Benchmark via Helm
+
+All defaults are the same as in the YAML installation method.
+Use the following steps to prep for installation:
+```bash
+# add helm repo
+helm repo add yasker-benchmark https://raw.githubusercontent.com/yasker/benchmark/main/helm
+
+# create namespace for needed resources
+kubectl create ns fio-benchmarks
+```
+
+To deploy with all defaults:
+```bash
+helm upgrade --install fio-benchmark-default \
+  yasker-benchmark/fio-standalone \
+  -n fio-benchmarks \
+  --version v0.1.0
+```
+
+To customize the installation, create a values.yaml or use `--set` flags:
+```bash
+# --set flags
+helm upgrade --install fio-benchmark-a \
+  yasker-benchmark/fio-standalone \
+  -n fio-benchmarks \
+  --version v0.1.0 \
+  --set pvc.size=11Gi \
+  --set benchmark.size=10G
```

Comment:
Can we automatically set PVC size to benchmark size * 1.1? Not sure if the chart can do that though.

Reply:
Good call, will update templates and values